editor/editor_node.cpp

```diff
@@ -6418,6 +6418,11 @@ EditorNode::EditorNode() {
 	p->add_icon_shortcut(gui_base->get_theme_icon(SNAME("HelpSearch"), SNAME("EditorIcons")), ED_SHORTCUT_AND_COMMAND("editor/editor_help", TTR("Search Help"), KEY_MASK_ALT | KEY_SPACE), HELP_SEARCH);
 #else
 	p->add_icon_shortcut(gui_base->get_theme_icon(SNAME("HelpSearch"), SNAME("EditorIcons")), ED_SHORTCUT_AND_COMMAND("editor/editor_help", TTR("Search Help"), KEY_F1), HELP_SEARCH);
+#endif
+#ifdef OSX_ENABLED
+	p->add_icon_shortcut(gui_base->get_theme_icon(SNAME("Search"), SNAME("EditorIcons")), ED_SHORTCUT("editor/command_palette", TTR("Open Command Palette"), KEY_MASK_CMD | KEY_MASK_SHIFT | KEY_P), HELP_COMMAND_PALETTE);
+#else
+	p->add_icon_shortcut(gui_base->get_theme_icon(SNAME("Search"), SNAME("EditorIcons")), ED_SHORTCUT("editor/command_palette", TTR("Open Command Palette"), KEY_MASK_CTRL | KEY_MASK_SHIFT | KEY_P), HELP_COMMAND_PALETTE);
```

Comment:
This shouldn't be necessary. Command = Control on windows, so you should just be able to set it to `KEY_MASK_CMD | KEY_MASK_SHIFT | KEY_P` and it should work on both operating systems.

Reply:
okay